include/engine/routing_algorithms/routing_base.hpp

```diff
@@ -484,6 +484,69 @@ template <class DataFacadeT, class Derived> class BasicRoutingInterface
         unpacked_path.emplace_back(t);
     }
 
+    /**
+     * A duplicate of the above `UnpackEdge` function, but returning full EdgeData
+     * objects for the unpacked path. Used in the tile plugin to find outgoing
+     * edges from a given turn.
+     */
+
```

Comment:
As already said in your comment, this feel like unnecessary duplication of the unpacking routine.
We can find the `EdgeID` for two vertices s/t.
With the `EdgeID`, you can get access to the path data.
I would prefer if we not add an additional `unpack` routine but use the existing one and transform the result into EdgeData at the location it is needed.